.env.example

```diff
@@ -26,6 +26,9 @@ GREENHOUSE_API_TOKEN='greenhouse-api-token'
 
 REDIS_URL=redis://localhost:6379/0
 
+ADOBE_LAUNCH_URL='https://assets.adobedtm.com/54d668d6379a/4bfed685397d/launch-1b04eb013f27.min.js'
```

Comment:
We should not add real values to the .env.sample file.

Reply:
Removed the URL and added empty string.